Directory.Build.props

```diff
@@ -58,6 +58,7 @@
         - src/installer/pkg/sfx/bundle/shared-framework-distribution-template-x64.xml
         - src/installer/pkg/sfx/bundle/shared-framework-distribution-template-arm64.xml
         - src/tasks/MobileBuildTasks/Apple/AppleProject.cs
+        - dotnet/installer repo > src/redist/targets/GeneratePKG.targets
```

Comment:
Remove the unused `watchOSVersionMin` and `watchOS64_32VersionMin` and add them when needed.

Reply:
Done